lib/src/mock_document_snapshot.dart

```diff
@@ -39,4 +44,17 @@ class MockDocumentSnapshot extends Mock implements DocumentSnapshot {
 
   @override
   SnapshotMetadata get metadata => _metadata;
+
+  bool _isComposedKey(String key) {
```

Comment:
This function is nice! Later we could use it in other places like in https://github.com/atn832/cloud_firestore_mocks/blob/d8be0b9c9d522a0d78ebe2303bc5756c09e75951/lib/src/mock_document_reference.dart#L124 to make the code more readable.

Would you mind renaming this to `_isCompositeKey` for consistency with the rest of the codebase?